client/state/jetpack-connect/actions.js

```diff
@@ -195,11 +196,44 @@ export function retryAuth( url, attemptNumber ) {
 	};
 }
 
+export function createSocialAccount( socialInfo ) {
+	return dispatch => {
+		dispatch( recordTracksEvent( 'calypso_jpc_social_create_account' ) );
+
+		dispatch( { type: JETPACK_CONNECT_CREATE_ACCOUNT } );
+
+		/**
+		 * @TODO (sirreal) update to `jetpack-connect` when D11099-code lands
+		 *
+		 * The signup flow is required and affects some post signup activity.
+		 * `account` should be safe until patch lands.
+		 */
+		return createSocialUser( socialInfo, 'account' /* 'jetpack-connect' */ )( dispatch ).then(
+			( { username, bearerToken } ) => {
+				dispatch( recordTracksEvent( 'calypso_jpc_social_createaccount_success' ) );
```

Comment:
Seems inconsistent with https://github.com/Automattic/wp-calypso/pull/23483/files#diff-4082b3f22f0a998cdc936b484d8ea8f9R201 -- should probably either be `calypso_jpc_social_create_account_success` (underscore between `create` and `account`), or `calypso_jpc_create_social_account_success`.

Reply:
I'm trying to walk a fine line of matching existing events and adhering to the naming conventions (PCYsg-4S2-p2):

> `<source>_<context>_<optional-subcontext>_<action>_<optional-qualifier>`